models/README.md

```diff
@@ -7,48 +7,41 @@ in this directory. Please also specify most recent **validation loss** and numbe
 * ``sdss_autoencoder.ckpt``
 
 ```bash
-# Validation Loss = 781677.125
-# Epochs = 969
-poetry run bliss mode="train" model="galaxy_net" dataset="sdss_galaxies" optimizer="adam" \
-training.n_epochs=1001 training.trainer.check_val_every_n_epoch=10 dataset.kwargs.noise_factor=0.01 \
-dataset.kwargs.num_workers=5 training.trainer.checkpoint_callback=True
+# Validation Loss = 782252.688
+# Epochs = 999
+poetry run bliss +experiment=sdss_autoencoder
 ```
 
 * ``sdss_binary.ckpt``
 
 ```bash
-# Validation Loss = 6.571
-# Epochs = 359
+# Validation Loss = 7.472
+# Epochs = 439
 # Notes:
 # - 20/09/21: Slight increase since galaxy PSF exactly equals decoder PSF
-poetry run bliss mode="train" model="binary_sdss" dataset="default" \
-optimizer.kwargs.lr=1e-4 training.n_epochs=501 training.trainer.checkpoint_callback=True \
-model.decoder.kwargs.mean_sources=0.03 training.save_top_k=5
+# - 20/09/29: Slightly worse than the previous one (0.571)
```

Comment:
what is `0.571` ? what are the reported accuracies in tensorboard? Can you maybe paste the plot produced in tensorboard here?

Reply:
Sorry, this was probably me misreading. I'll delete this line.